client/python/test/conftest.py

```diff
@@ -228,3 +228,15 @@ def all_components():
             subclasses.append(subclass)
 
     return subclasses
+
+
+@pytest.fixture(autouse=True)
+def disable_telemetry(monkeypatch):
```

Comment:
Nit: Instead of an autouse fixture, which means that this would be run before every single test, what about setting up these environment variables in `pytest_configure` so that they run a single time before all of the tests?

Reply:
Agreed - done!

Reply:
Thanks @freddyaboulton! So now we don't need these two fixtures, right?